pkg/controller/infra/infra_controller.go

```diff
@@ -459,7 +459,7 @@ func (s *jobSyncStrategy) ProcessDeletedOwner(owner metav1.Object) error {
 
 	// Clear the finalizer for the cluster
 	finalizers.Delete(clusteroperator.FinalizerClusterOperator)
-	cluster.ObjectMeta.Finalizers = finalizers.List()
```

Comment:
Why is this being removed?

Reply:
My bad, been trying to adjust to vscode and I'm doing something that kills lines without noticing.